sacred/observers/s3_observer.py

```diff
@@ -0,0 +1,328 @@
+#!/usr/bin/env python
```

Comment:
I think we do not need to put this on the tops of files anymore, as we are now python 3 only and this file should never be executed from the command line.

Reply:
Sounds good, I'll remove it